app/controllers/main_controller.rb

```diff
@@ -189,14 +189,14 @@ def redirectit(release)
 
 
   def developer
-    #redirectit("developer")
-    #return
-    flash.now[:warn] = _("There is no openSUSE release in testing phase at the moment. <br/>" +
-        " If you want to use bleeding edge software, please use <a href='http://en.opensuse.org/Portal:Tumbleweed'>openSUSE Tumbleweed</a>.")
-    @exclude_debug = true
-    @include_home = 'false'
-    set_release("421")
-    render :template => "main/release"
+    redirectit("developer")
+    return
+#    flash.now[:warn] = _("There is no openSUSE release in testing phase at the moment. <br/>" +
+#        " If you want to use bleeding edge software, please use <a href='http://en.opensuse.org/Portal:Tumbleweed'>openSUSE Tumbleweed</a>.")
+#    @exclude_debug = true
+#    @include_home = 'false'
+#    set_release("421")
+#    render :template => "main/release"
```

Comment:
Why comment code instead of removing it? If you want to maintain the code I would suggest to use some `if false` or the equivalent logic in Ruby